src/app/systelab-components/datepicker/datepicker.component.ts

```diff
@@ -237,7 +242,30 @@ export class Datepicker implements OnInit, AfterViewInit, DoCheck, OnDestroy {
 		}
 	}
 
-	public onKeyDown(event: KeyboardEvent) {
+	public formatDate(date: string ): string {
+		let dateTmp: string = date.trim();
+
+		if (dateTmp != null) {
+			if ((dateTmp.length === 8 || dateTmp.length === 10)
+				&& (dateTmp.lastIndexOf('/')  > 0 || dateTmp.lastIndexOf('-') > 0 || dateTmp.lastIndexOf('.') > 0)) {
+				dateTmp = dateTmp.replace('/', '')
+					.replace('-', '')
+					.replace('.', '');
+			}
```

Comment:
Be careful because it replace only the first instance

Reply:
Thanks a lot for this comment. I didn't know this in typescript.

Reply:
Thanks, I have already found a solution and put it into the PR